readme.md

```diff
@@ -39,13 +39,13 @@ camelcaseKeys(commandLineArguments);
 
 #### input
 
-Type: `object | object[]`
+Type: `Record<string, unknown> | any[]`
```

Comment:
Shouldn't the TypeScript types be this?

```suggestion
Type: `Record<string, unknown> | Record<string, unknown>[]`
```

Reply:
Yes, I think you are right.
I have included a fix to update the `index.d.ts` as well.